### PR TITLE
Ludvig

### PR DESCRIPTION
**Pull request by Luxwor**
Some shit

### CODE REVIEW

**Comment by Luxwor** [comment]
I should really not have pushed

**Comment by Luxwor** [comment]
Mabel I should have. Just pushing with an untested discord link that still runs.

**Comment by Luxwor** [comment]
issue 598 is not finished but everything else is good according to me

File: color.cpp [hunk 1]
```diff
@@ -198,22 +208,19 @@ void test_color()
...
 
-//#define FIX_ISSUE_460
-#ifdef FIX_ISSUE_460
   // #460 Ready-made black and white colors are available
   {
     const color black = create_black_color();
-    assert(get_redness(black) == 255);
-    assert(get_greenness(black) == 255);
-    assert(get_blueness(black) == 255);
+    assert(get_redness(black) == 0);
```
Comment (richelbilderbeek) [inline color.cpp:214]: Simple, thus good tests :+1:

**Review by richelbilderbeek — APPROVED** [review]
Try to give a proper message next time. I reviewed regardless as the PR was small and its a clear step forward.

**Pull request closed without merging**